scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,85 @@
+                                                 ledger_max_write_ledger_entries=Uint32(200),
+                                                 ledger_max_write_bytes=Uint32(1024000),
+                                                 tx_max_read_ledger_entries=Uint32(30),
+                                                 tx_max_read_bytes=Uint32(133120), # 130 kb
+                                                 tx_max_write_ledger_entries=Uint32(20),
+                                                 tx_max_write_bytes=Uint32(66560), # 65 kb 
+                                                 fee_read_ledger_entry=Int64(20000),
+                                                 fee_write_ledger_entry=Int64(30000),
+                                                 fee_read1_kb=Int64(1000),
+                                                 fee_write1_kb=Int64(1000),
+                                                 bucket_list_size_bytes=Int64(32212254720), # 30 * 1024 * 1024 * 1024 for 30 GB
+                                                 bucket_list_fee_rate_low=Int64(1000),
+                                                 bucket_list_fee_rate_high=Int64(10000),
+                                                 bucket_list_growth_factor=Uint32(1))
+
+    contract_ledger_cost_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_LEDGER_COST_V0,
+        contract_ledger_cost = contract_ledger_cost_settings)        
+
+    contract_historical_data_settings = ConfigSettingContractHistoricalDataV0(fee_historical1_kb=Int64(5000))
+
+    contract_historical_data_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_HISTORICAL_DATA_V0,
+        contract_historical_data = contract_historical_data_settings)    
+
+    contract_meta_data_settings = ConfigSettingContractMetaDataV0(tx_max_extended_meta_data_size_bytes=Uint32(2048), # 2 kb
+                                                                        fee_extended_meta_data1_kb=Int64(300))  
+
+    contract_meta_data_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_META_DATA_V0,
+        contract_meta_data = contract_meta_data_settings)    
+
+    contract_bandwidth_settings = ConfigSettingContractBandwidthV0(ledger_max_propagate_size_bytes=Uint32(1024000), # 100 kb
+                                                                        tx_max_size_bytes=Uint32(71620), # 70 kb
+                                                                        fee_propagate_data1_kb=Int64(2000))   
+
+    contract_bandwidth_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_BANDWIDTH_V0,
+        contract_bandwidth = contract_bandwidth_settings)    
+    
+    contract_data_entry_size = Uint32(65536) # 64 kb
+
+    contract_data_entry_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_DATA_ENTRY_SIZE_BYTES,
```

Comment:
Just to double check, we don't need to update `CONFIG_SETTING_CONTRACT_DATA_KEY_SIZE_BYTES`?

Reply:
I didn't find KEY size in the phased rollout [doc](https://docs.google.com/document/d/1RGFXy6gvzPUQwOtkcZrtvltjMBeqskwfZ__wcOCIXW8/edit) so left it out.

Reply:
Yeah, I don't think we need to increase the key size. We could reduce it to maybe 200 bytes, but I'm not really sure if that matters much.